.env-sample

```diff
@@ -0,0 +1,3 @@
+SNYK_TOKEN=replaceme
+COVERALLS_REPO_TOKEN=replaceme
+MONGO_IMAGE_TAG=replaceme with the latest tag from https://hub.docker.com/r/nhsuk/mongodb-pharmacy/tags/
```

Comment:
Using default variable values and getting a tagged version of the mongo image as `latest` means this can be removed